roles/variables/vars/main.yml

```diff
@@ -228,11 +202,11 @@ kafka_controller_properties:
     enabled: "{{ kafka_controller_metrics_reporter_enabled|bool }}"
     properties:
       metric.reporters: io.confluent.metrics.reporter.ConfluentMetricsReporter
-      confluent.metrics.reporter.bootstrap.servers: "{{ groups['kafka_controller'] | default(['localhost']) | confluent.platform.resolve_hostnames(hostvars) | join(':' + kafka_controller_listeners['controller']['port']|string + ',') }}:{{kafka_controller_listeners['controller']['port']}}"
-      confluent.metrics.reporter.topic.replicas: "{{kafka_controller_default_internal_replication_factor}}"
+      confluent.metrics.reporter.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | confluent.platform.resolve_hostnames(hostvars) | join(':' + kafka_broker_listeners['internal']['port']|string + ',') }}:{{kafka_broker_listeners['internal']['port']}}"
```

Comment:
why are we using broker here?

Reply:
The controller will report its metrics to the broker. As Ron mentioned, the controller does not have/hosts any topic where we could send/safe metrics.